spec.bs

```diff
@@ -497,6 +502,25 @@ The specification adds the following steps to the [=HTTP fetch=] algorithm, befo
 1. If |redeem response result| is a [=network error=], return |issue response result|.
 
 
+Integration with XMLHttpRequest {#xhr}
+====================================
+
+Modifications to Request {#xhr-request}
+---------------------------------------------------------------
+Refer to [[#fetch-request]].
+
+The <dfn><code>setPrivateToken()</code></dfn> method {#xhr-set-private-token}
+---------------------------------------------------------------
+This specification introduces a new [=setPrivateToken()=] that accepts a {{PrivateToken}} object specifying the {{OperationType}} to execute against the request. It must do the following:
```

Comment:
Q: Is the PrivateToken object identical to the struct used for the fetch spec?

Reply:
yes